apel/db/backends/mysql.py

```diff
@@ -293,6 +294,29 @@ def check_duplicate_sites(self):
                 self.db.rollback()
             raise
 
+    def summarise_storage(self):
+        '''
+        Calls the DailySummary stored procedure
+        '''
+        try:
+            # prevent MySQLdb from raising
+            # 'MySQL server has gone' exception
+            self._mysql_reconnect()
+
+            c = self.db.cursor()
+
+            log.info("Summarising storage records...")
+            c.callproc(self._summarise_storage_proc, ())
```

Comment:
I realise it's in the other similar methods, but what's the point of the empty brackets as an argument?

Reply:
You are right. In our case, Passing an empty args() to proc is not needed now. Calling the proc directly now. Done